tests/output/pjs/output_test.js

```diff
@@ -637,11 +637,27 @@ describe("Scratchpad Output Exec", function() {
     runTest({
         title: "Program functions exist",
         code: function() {
-            var x = Program.settings();
+            debug("start");
```

Comment:
whoopsies... that was from when I trying to diagnose the infinite loop issue below.   I'll clean that up.